core/src/main/java/io/questdb/griffin/SqlCodeGenerator.java

```diff
@@ -5564,7 +5564,7 @@ private void guardAgainstFillWithKeyedGroupBy(QueryModel model, ArrayColumnTypes
             return;
         }
 
-        if (curr.getFillValues().size() == 1 && Chars.equals(curr.getFillValues().getQuick(0).token, "none")) {
+        if (curr.getFillValues().size() == 1 && SqlKeywords.isNoneKeyword(curr.getFillValues().getQuick(0).token)) {
```

Comment:
nit: there is also `Chars.equals(ast.token, "timestamp_floor")` check in this class. Would be nice to make it case-insensitive too.